test/integration/StreamrClient.spec.js

```diff
@@ -22,16 +22,14 @@ describe('StreamrClient', function () {
     })
 
     before(() => Promise.all([ // TODO: Figure out how to handle this when setting up a CI
-        fetch(`http://${engineAndEditor}`),
-        fetch(`http://${dataApi}`),
+        axios.get(`http://${engineAndEditor}`),
+        axios.get(`http://${dataApi}`),
     ])
         .catch((e) => {
-            if (e.errno === 'ENOTFOUND' || e.errno === 'ECONNREFUSED') {
+            if (e.code === 'ENOTFOUND' || e.code === 'ECONNREFUSED') {
```

Comment:
are you sure these codes are right? when I run:

```
axios.get('http://httpstat.us/404').then(() => console.log('ok'), (e) => console.error(e.code)
```

the code is `undefined`.

Reply:
![](https://www.dropbox.com/s/8cwb7tk48mmctwe/Screenshot%202018-06-12%2011.58.52.png?raw=1)

At least when running the code in Node without `data-api` running in the background there is. But you're correct that when running in browser it's hard to get it to show the field (=didn't succeed)